combustion.service

```diff
@@ -12,7 +12,8 @@ Requires=combustion-prepare.service
 After=combustion-prepare.service
 
 # Optionally make network available
-After=network.target
+Wants=network-online.target
```

Comment:
TODO: Does this pull in nm-initrd.service even without `combustion: network`? Might have to enable `network-online.target` in combustion prepare dynamically if so.

Reply:
Shouldn't be an issue, both NM services have `ConditionPathExists=/run/NetworkManager/initrd/neednet`.